client.js

```diff
@@ -24,50 +29,140 @@ window.addEventListener('keydown', function (ev) {
   var ctrlW = ev.ctrlKey && w
   var cmdW = ev.metaKey && w
 
-  if (esc || ctrlW || cmdW) remote.getCurrentWindow().close()
+  if (esc || ctrlW || cmdW) currentWindow.close()
 })
 
 var zoom = require('./zoom')(conf.zoom)
 
 // menu
-var vmdSubmenu = [
-  { label: 'Quit', accelerator: 'CmdOrCtrl+Q', click: function () { remote.require('app').quit() } }
-]
+var vmdSubmenu = [{
+  label: 'Quit',
+  accelerator: 'CmdOrCtrl+Q',
+  click: function () {
+    remote.require('app').quit()
+  }
+}]
 
 if (process.platform === 'darwin') {
-  vmdSubmenu = [
-    { label: 'About vmd', selector: 'orderFrontStandardAboutPanel:' },
-    { type: 'separator' }
-  ].concat(vmdSubmenu)
+  vmdSubmenu = [{
+    label: 'About vmd',
+    selector: 'orderFrontStandardAboutPanel:'
+  }, {
+    type: 'separator'
+  }].concat(vmdSubmenu)
 }
 
-var template = [
-  {
-    label: 'vmd',
-    submenu: vmdSubmenu
-  },
-  {
-    label: 'File',
-    submenu: [
-      { label: 'Print', accelerator: 'CmdOrCtrl+P', click: function () { window.print() } }
-    ]
-  },
-  {
-    label: 'Edit',
-    submenu: [
-        { label: 'Copy', accelerator: 'CmdOrCtrl+C', click: function () { document.execCommand('copy') } },
-        { label: 'Select All', accelerator: 'CmdOrCtrl+A', click: function () { document.execCommand('selectAll') } }
-    ]
-  },
-  {
-    label: 'View',
-    submenu: [
-      { label: 'Zoom In', accelerator: 'CmdOrCtrl+Plus', click: function () { zoom.zoomIn() } },
-      { label: 'Zoom Out', accelerator: 'CmdOrCtrl+-', click: function () { zoom.zoomOut() } },
-      { label: 'Reset Zoom', accelerator: 'CmdOrCtrl+0', click: function () { zoom.reset() } }
-    ]
+var template = [{
+  label: 'vmd',
+  submenu: vmdSubmenu
+}, {
+  label: 'File',
+  submenu: [{
+    label: 'Print',
+    accelerator: 'CmdOrCtrl+P',
+    click: function () {
+      window.print()
+    }
+  }]
+}, {
+  label: 'Edit',
+  submenu: [{
+    label: 'Copy',
+    accelerator: 'CmdOrCtrl+C',
+    click: function () {
+      document.execCommand('copy')
+    }
+  }, {
+    label: 'Select All',
+    accelerator: 'CmdOrCtrl+A',
+    click: function () {
+      document.execCommand('selectAll')
+    }
+  }]
+}, {
+  label: 'View',
+  submenu: [{
+    label: 'Zoom In',
+    accelerator: 'CmdOrCtrl+Plus',
+    click: function () {
+      zoom.zoomIn()
+    }
+  }, {
+    label: 'Zoom Out',
+    accelerator: 'CmdOrCtrl+-',
+    click: function () {
+      zoom.zoomOut()
+    }
+  }, {
+    label: 'Reset Zoom',
+    accelerator: 'CmdOrCtrl+0',
+    click: function () {
+      zoom.reset()
+    }
+  }, {
+    label: 'Toggle Developer Tools',
+    accelerator: (function () {
+      if (process.platform === 'darwin') {
+        return 'Alt+Command+I'
+      } else {
+        return 'Ctrl+Shift+I'
+      }
+    })(),
+    click: function (item, focusedWindow) {
+      if (focusedWindow) {
+        focusedWindow.toggleDevTools()
+      }
+    }
+  }]
+}]
```

Comment:
@yisibl Yeah, I agree that writing objects on multiple lines is better and produces better diffs and it makes a difference especially when objects have more than one or two keys.  
But my question about the indentation and formatting was more about why this:

```js
var obj = [{
  k1: 'v1',
  k2: 'v2'
}, {
  k1: 'v1',
  k2: 'v2'
}]
```

over this:

```js
var obj = [
  {
    k1: 'v1',
    k2: 'v2'
  },
  {
    k1: 'v1',
    k2: 'v2'
  }
]
```

?